movement/filtering.py

```diff
@@ -90,6 +93,8 @@ def interpolate_over_time(
     print_report : bool
         Whether to print a report on the number of NaNs in the dataset
         before and after interpolation. Default is ``False``.
+    fill_value : float, optional
```

Comment:
This input argument is only defined in the docstring: it does not appear in the function signature or in the function body.

It would either need to be removed, or properly implemented. As it is, it will show up in the API reference but it is not actually implemented, which would be very confusing for our users.

Reply:
Additionally, the `constant` method doesn't seem to be supported for the `method` argument